plugins/inputs/mailchimp/mailchimp.go

```diff
@@ -11,9 +11,9 @@ import (
 type MailChimp struct {
 	api *ChimpAPI
 
-	ApiKey     string
+	APIKey     string
 	DaysOld    int
-	CampaignId string
+	CampaignID string
```

Comment:
These might be exposed to `toml`. Are you sure that the `toml` config parser still does the same as before?

Reply:
Right, missed adding `toml` tags in this file. Fixed.